src/components/search/FullSiteSearch.jsx

```diff
@@ -0,0 +1,131 @@
+/** @jsxImportSource @emotion/react */
+import tw from "twin.macro";
+import React from "react";
+import { XIcon } from "../icons/Icons";
+import axios from "axios";
+
+const SearchResultItem = ({ placeTitle, placeLink, dealTitle, className }) => {
+  return (
+    <div className={className} tw="p-4">
+      <a href={`/place/${placeLink}`}>
+        <h3 tw="font-bold text-xl">{placeTitle}</h3>
+        <p>{dealTitle}</p>
+      </a>
+    </div>
+  );
+};
+export const FullSiteSearch = ({ showSearch, onSearchDismiss }) => {
+  const [searchResults, setSearchResults] = React.useState(null);
+  const [searchTerm, setSearchTerm] = React.useState("");
+
+  const fetchData = React.useCallback(async () => {
+    if (searchTerm.length < 2) {
+      setSearchResults(null);
+      return;
+    }
+    try {
+      const res = await axios.get(
+        `${process.env.REACT_APP_API_URL}/api/deal/search?text=${searchTerm}`
+      );
+      setSearchResults(res.data);
+    } catch (e) {
+      console.error("Error fetching data", e);
+      setSearchResults(null);
+    }
+  }, [searchTerm]);
+
+  React.useEffect(() => {
+    (async () => {
+      await fetchData();
+    })();
+  }, [fetchData]);
+
+  return (
+    <>
+      {showSearch ? (
```

Comment:
![Screen Shot 2023-07-15 at 1 02 50 PM](https://github.com/alexsaltstein/HobokenEventsWebApp/assets/25209141/6920c643-4aaf-4b27-8018-8b58af05a5ac)


From a mobile perspective.  I believe that the search should span the entirety of the window. feels very dialog/modally from this size when we want it to be more of a popover element. I also think maybe change the text to something along the lines of "Start typing to find your desired Happy hour" or something that's not explicitly our if check lol.